setup.py

```diff
@@ -5,6 +5,14 @@
 
 from setuptools import setup
 
+install_requires = ['catkin_pkg', 'PyYAML']
+
+if (
+    sys.version_info[0] > 3 or
+    sys.version_info[0] == 3 and sys.version_info[1] >= 8
```

Comment:
Is the version of python used to call `setup.py` guaranteed to be the same as the version of python on the target system? Does that mean during release the `setup.py` invoked multiple times, once for each target platform?

Reply:
I agree with your concern. The first commit probably doesn't work.

Please see 9627024db19a3832be7ab2058e7811bbd9bf72e4 for an alternative approach.